src/test/java/com/wikia/webdriver/common/contentpatterns/InteractiveMapsContent.java

```diff
@@ -20,31 +19,27 @@
   public static final int[] MAP_TO_DELETE_AND_RESTORE = {4258, 71, 72};
   public static final String ASSOCIATED_ARTICLE_NAME = "File:400x200.png";
   public static final String TEMPLATE_NAME_TO_SEARCH_SHOULD_BE_FOUND = "te";
-  public static final String
-      TEMPLATE_NAME_TO_SEARCH_SHOULD_NOT_BE_FOUND =
+  public static final String TEMPLATE_NAME_TO_SEARCH_SHOULD_NOT_BE_FOUND =
       "1234567890123123123123213";
-  public static String TEMPLATE_NAME;
   public static final String LEARN_MORE_LINK = "http://maps.wikia.com";
   public static final String MAP_NAME = "testingMapName";
   public static final String PIN_TYPE_NAME = "testingPinType";
   public static final String PIN_NAME = "testingPinName";
   public static final String MAP_DELETED_SUCCESSFUL = "Map was successfully deleted!";
-  public static final String
-      MAP_DELETE_ERROR =
+  public static final String MAP_DELETE_ERROR =
       "You are not allowed to execute the action you have requested.";
   public static final String MAP_DELETED_MSG = "This map is deleted";
   public static final String EXTERNAL_LINK = "http://www.gazeta.pl/0,0.html";
-  public static final String
-      ARTICLE_WHICH_DOES_NOT_EXIST =
+  public static final String ARTICLE_WHICH_DOES_NOT_EXIST =
       "asdkljaskldjaskldjaskldjaskldjaskldjaskldjaskldjaskljdaskl";
-  public static final String
-      ASSOCIATED_ARTICLE_PLACEHOLDER =
+  public static final String ASSOCIATED_ARTICLE_PLACEHOLDER =
       "Related article name or external URL";
-  public static final String
-      ARTICLE_NOT_EXIST_ERROR =
+  public static final String ARTICLE_NOT_EXIST_ERROR =
       "Error: either there is no %articlename% article on this wikia, or the URL entered is invalid.";
   public static final String ESCAPED_FRAGMENT_MAP_ID = "69";
-  public static final String MOBILE_APPS_MAP = "http://maps.wikia-services.com/api/v1/render/77/?hideAttr=1&isEmbeddedInWikiaApp&uselang=en&noexternals=1";
+  public static final String MOBILE_APPS_MAP =
+      "http://maps.wikia-services.com/api/v1/render/77/?hideAttr=1&isEmbeddedInWikiaApp&uselang=en&noexternals=1";
+  public static String TEMPLATE_NAME;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make TEMPLATE_NAME a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "public static TEMPLATE_NAME" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)